linux-tkg/customization.cfg

```diff
@@ -0,0 +1,192 @@
+
+# CPU scheduler - Options are "upds" (TkG's Undead PDS), "pds", "bmq", "muqss" or "cfs"
+# "upds" is the recommended option for gaming
+_cpusched=""
+
+# Compiler to use - Options are "gcc" or "llvm".
+# For advanced users.
+_compiler=""
+
+# CPU sched_yield_type - Choose what sort of yield sched_yield will perform
+# For PDS and MuQSS: 0: No yield. (Recommended option for gaming on PDS and MuQSS)
+#                    1: Yield only to better priority/deadline tasks. (Default - can be unstable with PDS on some platforms)
+#                    2: Expire timeslice and recalculate deadline. (Usually the slowest option for PDS and MuQSS, not recommended)
+# For BMQ:           0: No yield.
+#                    1: Deboost and requeue task. (Default)
+#                    2: Set rq skip task.
+_sched_yield_type=""
+
+# Round Robin interval is the longest duration two tasks with the same nice level will be delayed for. When CPU time is requested by a task, it receives a time slice equal
+# to the rr_interval in addition to a virtual deadline. When using yield_type 2, a low value can help offset the disadvantages of rescheduling a process that has yielded.
+# MuQSS default: 6ms"
+# PDS default: 4ms"
+# BMQ default: 2ms"
+# Set to "1" for 2ms, "2" for 4ms, "3" for 6ms, "4" for 8ms, or "default" to keep the chosen scheduler defaults.
+_rr_interval=""
+
+# Set to "true" to disable FUNCTION_TRACER/GRAPH_TRACER, lowering overhead but limiting debugging and analyzing of kernel functions - Kernel default is "false"
+_ftracedisable="false"
+
+# Set to "true" to disable NUMA, lowering overhead, but breaking CUDA/NvEnc on Nvidia equipped systems - Kernel default is "false"
+_numadisable="false"
+
+# Set to "true" to enable misc additions - May contain temporary fixes pending upstream or changes that can break on non-Arch - Kernel default is "true"
+_misc_adds="true"
+
+# Set to "1" to use CattaRappa mode (enabling full tickless), "2" for tickless idle only, or "0" for periodic ticks.
+# Full tickless can give higher performances in various cases but, depending on hardware, lower consistency. Just tickless idle can perform better on some platforms (mostly AMD based).
+_tickless=""
+
+# Setting this to to "true" can improve latency on PDS (at the cost of throughput) and improve throughput on other schedulers (at the cost of latency) - Can improve VMs performance - Kernel default is "false"
+_voluntary_preempt=""
+
+# Set to "true" to enable Device Tree and Open Firmware support. If you don't know about it, you don't need it - Default is "false"
+_OFenable="false"
+
+# Set to "true" to use ACS override patch - https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Bypassing_the_IOMMU_groups_.28ACS_override_patch.29 - Kernel default is "false"
+_acs_override=""
+
+# Set to "true" to add Bcache filesystem support. You'll have to install bcachefs-tools-git from AUR for utilities - https://bcachefs.org/ - If in doubt, set to "false"
+#_bcachefs=""
```

Comment:
This should be uncommented as it's selectively disabled depending on kernel versions